### PR TITLE
Insert error code sql migrations added

### PR DESCRIPTION
**Pull request by danishjamal104**

### CODE REVIEW

**Comment by danishjamal104** [comment]
@avikganguly01 and @SubhamPramanik please review and merge :).

File: src/main/resources/sql/migrations/tenant/V32__insert_errorcode.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    'The transaction is being processed',
+    '500.001.1001',
+    true
+);
+
+INSERT INTO `errorcode` (`TRANSACTION_TYPE`, `ERROR_MESSAGE`, `ERROR_CODE`, `RECOVERABLE`)
+VALUES (
+    'COLLECTION-MPESA',
+    'DS timeout.',
+    '1037',
```
Comment (avikganguly01) [inline src/main/resources/sql/migrations/tenant/V32__insert_errorcode.sql:13]: Isnt ds timeout and 1037 the same thing?

Can you also include errors for error.contains network and connection - the 2 most common errors?

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

File: src/main/resources/sql/migrations/tenant/V32__insert_errorcode.sql (hunk 1 above)
Comment (danishjamal104) [reply src/main/resources/sql/migrations/tenant/V32__insert_errorcode.sql:13]: no they are not same. One is `transactionMessage` and another one is `errorCode`.

Comment (danishjamal104) [reply src/main/resources/sql/migrations/tenant/V32__insert_errorcode.sql:13]: didn't get the second question can you elaborate on it.

**Pull request merged**